README.md

```diff
@@ -1,5 +1,57 @@
 # stream-freshdesk
 
+A data stream generator for the Freshdesk REST API, written in python 3.
```

Comment:
- I think we should mention that this is a singer tap and produces JSON-formatted data following the singer spec (with appropriate links)
 - Lets be as descriptive as possible here for prospective users:
   - Pulls raw data from Freshdesk's REST API (link to their docs?)
   - Includes these resources: tickets, agents, ...
   - Works incrementally
   - ...